src/stores/topAccounts.js

```diff
@@ -26,8 +28,35 @@ export const useTopAccountsStore = defineStore('topAccounts', () => {
     rawTopAccounts.value = data
   }
 
+  async function fetchActiveAccountsCount() {
+    activeAccountsCount.value = null
+    const { data } = await axios.get(`${MIDDLEWARE_URL}/v3/stats/active-accounts?limit=1`)
+    activeAccountsCount.value = data.data[0].count
+  }
+
+  async function fetchTotalAccountsCount() {
+    totalAccountsCount.value = null
+    const { data } = await axios.get(`${MIDDLEWARE_URL}/v3/stats/total-accounts?interval_by=month&limit=100`)
+    // todo ask mdw for endpoint
+    totalAccountsCount.value = data.data.reduce((total, item) => total + parseInt(item.count), 0)
```

Comment:
It works based on manually summing the accounts from time fragments. Which is ok for now, not a clean solution.
Clean solution followup https://github.com/aeternity/aescan/issues/1009